docs/git/fixing-commits.md

```diff
@@ -40,6 +40,30 @@ Sometimes, you want to make one commit out of a bunch of commits. To do this,
 1. `git rebase -i HEAD~n` where `n` is the number of commits you are interested in
 2. Reorder the lines containing the commits and save
 
+## Changing code in an older commit
+
+If your pull request has multiple commits, a reviewer may request changes that require
+you to make changes to an older commit than the latest one. This tends to be more tedious
+than just amending the last commit and there can be various approaches, but the most efficient
+is by using `git commit --fixup`:
+
+1. Make your changes to the files and stage them, as usual.
```

Comment:
I think it's usually more efficient to start with `git rebase -i` and editing the older commit -- that's what I always do, because it avoids risk that changes made on top of the last change merge conflict with intermediate commits.